### DISCUSSION TITLE
Release v1.12.8 / 5.67.8

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#3588] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### Release Notes

This is a maintenance release, it fixes many issues, for a full list of changes and fixes please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1128--5678---2024-01-31).

### Known Issue
- Symbolic link resolution does not work correctly. Please install build [1.12.9](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.12.9).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.12.8'>Release v1.12.8 / 5.67.8</a>.</em>

**Comment by BilleBarrett** [comment]
Was this update put on the distribution list, so my clients will be updated?
my auto-update software still says 1.12.8 not 8b

**Reply by bastik-1001** [reply]
Please wait for 1.12.9 to be distributed.

**Comment by BilleBarrett** [comment]
could someone please tell me why my downloads are not giving me notice that they need to be recovered.
I can't seem to figure out where the setting is, this happens every time there is an update

**Reply by bastik-1001** [reply]
You can enable or disable file recovery here:

Sandbox Options > File Options > [File Recovery] > "Enable Immediate Recovery prompt to be able to recover files as soon as they are created."

**Comment by BilleBarrett** [comment]
I already have it and have installed on all my clients 

 

-----------------------------------------------------------------------------------------------------------------


The bridge and the wall are made with the same material

The Bridge joins the people and the Wall divides the people

Choose the Right One

Remote Desktop Services 
(336) 988-0891    
 ***@***.***> ***@***.***

**Comment by BilleBarrett** [comment]
I have it already set and for some reason it started working again





-----------------------------------------------------------------------------------------------------------------


The bridge and the wall are made with the same material

The Bridge joins the people and the Wall divides the people

Choose the Right One

Remote Desktop Services 
(336) 988-0891    
 ***@***.***> ***@***.***
